app/views/questionnaire.py

```diff
@@ -125,6 +125,8 @@ def save_questionnaire_store_wrapper(*args, **kwargs):
 @full_routing_path_required
 def get_block(routing_path, eq_id, form_type, collection_id, group_id, group_instance, block_id):  # pylint: disable=unused-argument,too-many-locals
     current_location = Location(group_id, group_instance, block_id)
+    metadata = get_metadata(current_user)
+    schema = load_schema_from_metadata(metadata)
```

Comment:
worth considering this as a wrapper? like we do with `@full_routing_path_required`?
given we repeat these lines in most of the views.

Reply:
It is for sure, I've suggested that we use pytest fixtures for most of these repeated helpers here: https://trello.com/c/zuIZkU2N/674-make-more-use-of-pytest-fixtures-for-simplification-and-speeding-up-of-tests

Reply:
But I've not included it here because this was already getting pretty big!

Reply:
I wasn't thinking specifically about pytest. i was looking more at the repeated lines in every view

Reply:
Id say it was a change related to the code changed here?

Reply:
Yeah, okay fair. I'll do that.